calendar-bundle/src/EventListener/SitemapListener.php

```diff
@@ -56,6 +65,10 @@ public function __invoke(SitemapEvent $event): void
                 continue;
             }
 
+            if ($isMember && $objCalendar->protected && !$this->security->isGranted(ContaoCorePermissions::MEMBER_IN_GROUPS, $objCalendar->groups)) {
```

Comment:
I don't think `$isMember` should be checked here, because there is also a guest checkbox. Not sure if we can ever exclude protected archives.

Reply:
We don't have the option _show only guests_ on news archives and calendars.